Fix mismatched array bounds for the parameters of function
rsVec_cross()'s declaration and definition.

--- a/src/rsMath.c
+++ b/src/rsMath.c
@@ -43,7 +43,7 @@ float rsVec_normalize (float *v)
 	return length;
 }
 
-void rsVec_cross (float *v, float vec1[4], float vec2[4])
+void rsVec_cross (float *v, float vec1[3], float vec2[3])
 {
 	v[0] = vec1[1] * vec2[2] - vec2[1] * vec1[2];
 	v[1] = vec1[2] * vec2[0] - vec2[2] * vec1[0];
